### DISCUSSION TITLE
HTTP2 Protocol

### DISCUSSION THREAD

**Discussion by CodingRuo** [nodejs/undici#2634] [Q&A]
Hi! I have a question, why wasn't the HTTP/2 standard used? Since it essentially offers more advantages than the old HTTP/1.1 protocol? What was the reason behind this in the design decision?

Thank you and best regards :)

**Comment by mcollina** [comment]
First, HTTP/2 is supported by behind a flag. 
Second, the answer to why we focused on HTTP/1.1 first is long and require a lot of context.
I'll write/record some longform content whenever I get the chance.
